src/main/kotlin/net/maizegenetics/phgv2/utils/GeneralUtilities.kt

```diff
@@ -29,3 +34,149 @@ fun inputStreamProcessing(tiledbList: BufferedInputStream): List<String> {
     }
     return samples
 }
+
```

Comment:
The tiledbList.close() above... Should that be closed by what ever created it?

Reply:
I believe it gets closed when the code runs without error, but am not sure it will be closed if there was an exception in the middle of processing.